frontend/src/components/Pages/InspectionPage/InspectionTable.tsx

```diff
@@ -22,161 +22,338 @@ const StyledIcon = styled(Icon)`
+                        <StyledCircle style={{ background: 'red' }} />
+                        {TranslateText('Not yet performed')}
+                    </StyledContent>
+                )
             } else {
-                status = inspection.missionDefinition.inspectionFrequency
-                    ? getInspectionStatus(inspection.deadline!)
-                    : TranslateText('No planned inspection')
-                lastCompleted = formatDateString(mission.lastRun.endTime!)
+                status = (
+                    <StyledContent>
+                        <StyledCircle style={{ background: 'green' }} />
+                        {TranslateText('No planned inspection')}
+                    </StyledContent>
+                )
             }
+            lastCompleted = TranslateText('Never')
+        } else {
+            status = inspection.missionDefinition.inspectionFrequency ? (
+                getInspectionStatus(inspection.deadline!)
+            ) : (
+                <StyledContent>
+                    <StyledCircle style={{ background: 'green' }} />
+                    {TranslateText('No planned inspection')}
+                </StyledContent>
+            )
+            lastCompleted = formatDateString(mission.lastRun.endTime!)
         }
+    }
 
-        return (
-            <Table.Row key={mission.id}>
-                <Table.Cell>{status}</Table.Cell>
-                <Table.Cell>
-                    <Typography
-                        link
-                        onClick={() => navigate(`${config.FRONTEND_BASE_ROUTE}/mission-definition/${mission.id}`)}
-                    >
-                        {mission.name}
-                    </Typography>
-                </Table.Cell>
-                <Table.Cell>{mission.area.areaName}</Table.Cell>
-                <Table.Cell>{mission.comment}</Table.Cell>
-                <Table.Cell>{lastCompleted}</Table.Cell>
-                <Table.Cell>{inspection.deadline ? inspection.deadline.toDateString() : ''}</Table.Cell>
-                <Table.Cell>
-                    <StyledIcon
-                        color={`${tokens.colors.interactive.focus.hex}`}
-                        name={Icons.AddOutlined}
-                        size={16}
-                        title={TranslateText('Add to queue')}
+    return (
+        <Table.Row key={mission.id}>
+            <Table.Cell>{status}</Table.Cell>
+            <Table.Cell>
+                <Typography
+                    link
+                    onClick={() => navigate(`${config.FRONTEND_BASE_ROUTE}/mission-definition/${mission.id}`)}
+                >
+                    {mission.name}
+                </Typography>
+            </Table.Cell>
+            <Table.Cell>{mission.comment}</Table.Cell>
+            <Table.Cell>{mission.area.areaName}</Table.Cell>
+            <Table.Cell>{lastCompleted}</Table.Cell>
+            <Table.Cell>{inspection.deadline ? inspection.deadline.toDateString() : ''}</Table.Cell>
```

Comment:
Should be formatted in a more similar way

Preferably also a bit more compact formatting as its very tight in pad mode